src/Analyzers/CSharp/Tests/UseImplicitOrExplicitType/UseExplicitTypeTests.cs

```diff
@@ -470,6 +476,9 @@ void Method(Program? x)
             await TestInRegularAndScriptAsync(before, after, options: ExplicitTypeExceptWhereApparent());
         }
 
+#if !CODE_STYLE // TODO: Skipped tests in CodeStyle layer depend on new Nullable APIs that are not available in CodeStyle layer.
```

Comment:
is it not posible do to this with a special Fact/Skip attribute?

Reply:
i looked at https://github.com/dotnet/roslyn/issues/41462.  it doesn't specifically mention the Nullable stuff.  can you explicitly add it to that issue?

Reply:
I added an explicit bullet item in https://github.com/dotnet/roslyn/issues/41462#issue-561271304:

> Revert #if CODE_STYLE directives added to product source and tests due for the scenario where certain new Nullability APIs are not yet available in CodeStyle layer. This might mean either moving the CodeStyle layer to new Microsoft.CodeAnalysis API OR adding reflection based lightup support in CodeStyle layer.

I think we need to discuss the right approach to address this at Monday's design meeting - whether to use reflection based lightup in CodeStyle layer OR moving the CodeStyle layer to latest Microsoft.CodeAnalysis version. I personally prefer the latter, but I know @sharwell prefers the former approach.

Reply:
> is it not posible do to this with a special Fact/Skip attribute?

I am really hoping to revert this soon - likely early next week once we decide on the above design question. @sharwell already has lightup based implementation that works, so it is just a matter of making the correct design decision and reverting these `#if CODE_STYLE` directives due to Nullable APIs.

Reply:
Sounds good.  As long as we're vigilant here, i'm ok with this. :)